src/explorer/SiteTreeItem.ts

```diff
@@ -68,7 +68,8 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
         const isSsl = this.site.hostNameSslStates.findIndex(value =>
             value.name === defaultHostName && value.sslState === `Enabled`);
         // tslint:disable-next-line:no-http-string
-        const uri = `${isSsl ? 'https://' : 'http://'}${defaultHostName}`;
+        const uri = `${isSsl !== -1 ? 'https://' : 'http://'}${defaultHostName}`;
```

Comment:
Rather than change this line, you could change isSsl from 'findIndex' to 'some', which returns a boolean and "Determines whether the specified callback function returns true for any element of an array."